src/App/constants.js

```diff
@@ -10,6 +12,8 @@ export const TEXT_LOCATION = {
   center: 'center',
 };
 
+export const DEFAULT_OUTPUT_DIRECTORY = path.join(os.homedir(), 'Bible Karaoke Videos');
```

Comment:
I wonder if we should distinguish between Windows and other platforms here for the default.  I find that Linux and Mac users tend to expect things to show up in their home directory, whereas Windows users expect to find files in 'Documents' or 'Pictures' or 'Videos' or 'Music' as these have direct links in the UI (the home directory does not).  So for Windows, I wonder if we should default to the Videos folder as the root.  Thoughts?
![image](https://user-images.githubusercontent.com/3444521/92355285-1f964780-f10e-11ea-8c79-5f7eea5ec736.png)

Reply:
@megahirt that's a good call. I've created issue #112 to track this improvement.